lib/rubycritic/cli/options.rb

```diff
@@ -40,6 +40,16 @@ def parse
             @suppress_ratings = true
           end
 
+          opts.on("-O ", "--open", [:'chrome', :firefox, :safari, :chromium],
```

Comment:
2 points:

1.) The quotation marks for

>> :'chrome'

are unnecessary and seem to be a remnant.

2.) Can we make the list of supported browsers a constant so that we can reference that here and [here](https://github.com/whitesmith/rubycritic/pull/100/files#diff-0710b2dc2bc4fa0cf5b4aa9a98870c48R12)